src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/UnsupportedSettingsHelper.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Wox.Plugin.Logger;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsSettings.Helper
+{
+    /// <summary>
+    /// Helper class to easier work with the version of the Windows OS
+    /// </summary>
+    internal static class UnsupportedSettingsHelper
+    {
+        private const string _keyPath = "HKEY_LOCAL_MACHINE\\SOFTWARE\\WOW6432Node\\Microsoft\\Windows NT\\CurrentVersion";
```

Comment:
This definitively not a good idea!! Will be a problem on x86 and arm64 in the future.

We need this here:
```suggestion
        private const string _keyPath = "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion";
```